src/components/Global/CrispIframe/index.tsx

```
@@ -0,0 +1,106 @@
+
+/**
+ * Shared Crisp iframe component with loading and error states
+ * DRY component used by both SupportDrawer and SupportPage
+ */
+export const CrispIframe = ({ crispProxyUrl, enabled = true }: CrispIframeProps) => {
+    const [componentError, setComponentError] = useState<string | null>(null)
+
+    // Defensive: Catch any hook errors
+    let hookState
+    try {
+        hookState = useCrispIframeReady(enabled)
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)